code/modules/power/reactor/rbmk.dm

```diff
@@ -0,0 +1,1011 @@
+#define RBMK_TEMPERATURE_CRITICAL 810 //At this point the entire ship/station is alerted to a meltdown. This may need altering
+#define RBMK_TEMPERATURE_MELTDOWN 900
+
+#define RBMK_NO_COOLANT_TOLERANCE 5 //How many process()ing ticks the reactor can sustain without coolant before slowly taking damage
+
+#define RBMK_PRESSURE_OPERATING 1000 //PSI
+#define RBMK_PRESSURE_CRITICAL 1469.59 //PSI
+
+#define RBMK_MAX_CRITICALITY 3 //No more criticality than N for now.
+
+//#define RBMK_POWER_FLAVOURISER 1000 //To turn those KWs into something usable
+#define RBMK_POWER_FLAVOURISER 8000 //To turn those KWs into something usable
+
+//Math. Lame.
+#define KPA_TO_PSI(A) (A/6.895)
+#define PSI_TO_KPA(A) (A*6.895)
+#define KELVIN_TO_CELSIUS(A) (A-273.15)
+#define MEGAWATTS /1e+6
+#define CELSIUS_TO_KELVIN(T_K)	((T_K) + T0C)
+
+//Reference: Heaters go up to 500K.
+//Hot plasmaburn: 14164.95 C.
+
+/**
+What is this?
+Moderators list (Not gonna keep this accurate forever):
+Fuel Type:
+Oxygen: Power production multiplier. Allows you to run a low plasma, high oxy mix, and still get a lot of power.
+Plasma: Power production gas. More plasma -> more power, but it enriches your fuel and makes the reactor much, much harder to control.
+Tritium: Extremely efficient power production gas. Will cause chernobyl if used improperly.
+Moderation Type:
+N2: Helps you regain control of the reaction by increasing control rod effectiveness, will massively boost the rad production of the reactor.
+CO2: Super effective shutdown gas for runaway reactions. MASSIVE RADIATION PENALTY!
+Pluoxium: Same as N2, but no cancer-rads!
+Permeability Type:
+BZ: Increases your reactor's ability to transfer its heat to the coolant, thus letting you cool it down faster (but your output will get hotter)
+Water Vapour: More efficient permeability modifier
+Hyper Noblium: Extremely efficient permeability increase. (10x as efficient as bz)
+Depletion type:
+Nitryl: When you need weapons grade plutonium yesterday. Causes your fuel to deplete much, much faster. Not a huge amount of use outside of sabotage.
+Sabotage:
+Meltdown:
+Flood reactor moderator with plasma, they won't be able to mitigate the reaction with control rods.
+Shut off coolant entirely. Raise control rods.
+Swap all fuel out with spent fuel, as it's way stronger.
+Blowout:
+Shut off exit valve for quick overpressure.
+Cause a pipefire in the coolant line (LETHAL).
+Tack heater onto coolant line (can also cause straight meltdown)
+Tips:
+Be careful to not exhaust your plasma supply. I recommend you DON'T max out the moderator input when youre running plasma + o2, or you're at a tangible risk of running out of those gasses from atmos.
+The reactor CHEWS through moderator. It does not do this slowly. Be very careful with that!
+*/
+
+//Remember kids. If the reactor itself is not physically powered by an APC, it cannot shove coolant in!
+GLOBAL_DATUM(rbmk_loop, /datum/looping_sound/rbmk_reactor)
```

Comment:
```suggestion
GLOBAL_DATUM(soundloop, /datum/looping_sound/rbmk_reactor)
```

Reply:
actually forget this, just rename the others to rbmk_loop

Reply:
it doesn't like it either way.

Reply:
Uh, don't add another var for it and, try using
```suggestion
GLOBAL_DATUM_INIT(rbmk_loop, /datum/looping_sound/rbmk_reactor)
```

Reply:
Also every instance of soundloop should be replaced with `GLOB.rmbk_loop`